js/src/jsonrpc/interfaces/parity.js

```diff
@@ -1674,9 +1674,9 @@ export default {
         example: {
           from: '0xb60e8dd61c5d32be8058bb8eb970870f07233155',
           to: '0xd46e8dd67c5d32be8058bb8eb970870f07244567',
-          gas: fromDecimal(30400),
-          gasPrice: fromDecimal(10000000000000),
-          value: fromDecimal(2441406250),
+          gas: 30400,
```

Comment:
I think he meant to convert the decimals into hex values so that the example is not using any functions (but I'm unsure though).